### PR TITLE
Add a section on inappropriate alcohol and drugs.

### PR DESCRIPTION
**Pull request by rduplain**
This came up during review in #13. The revision speaks for itself.

### CODE REVIEW

**Review by itruett — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -192,6 +193,13 @@ Unless a group or event identifies as political in nature, misconduct includes d
 Notwithstanding, discussions about _policies_ are generally acceptable. There is a difference between discussing policies in the context of their impact on the community and sharing reactionary comments to recent headlines. While it is acceptable to share reactions generally, reactions to political headlines or controversial topics are only accepted when a group, event, or space identifies as political in nature.
 
 
+### Inappropriate alcohol/drugs
+
+Notwithstanding acceptable discussion and activity involving responsibly-consumed alcohol and legal drugs (including caffeine), misconduct includes discussion or activity which involves excessive use of alcohol or any drug, humor about addiction, suggestions that addiction is acceptable or is not a problem, encouraging illegal drug use, and social pressure in use of any alcohol or drug.
```
Comment (Meekohi) [inline README.md:198]: This sentence structure is a bit hard to parse for me. "Notwithstanding Y, X includes [A, B, C]"

Suggest:
"Misconduct includes discussion or activity which involves excessive use of alcohol or any drug, humor about addiction, suggestions that addiction is acceptable or is not a problem, encouraging illegal drug use, and social pressure to use any alcohol or drug. Discussion and activity involving responsibly-consumed alcohol and legal drugs (including caffeine) is acceptable."

File: README.md [hunk 2]
```diff
@@ -192,6 +193,13 @@ Unless a group or event identifies as political in nature, misconduct includes d
...
+
+As with politics above, it is acceptable to discuss drug-related policies and to advocate for creating spaces in the community where participants can opt into discussions which risk drug-related discussion misconduct.
```
Comment (Meekohi) [inline README.md:200]: Personally I'd skip the "with politics above" -- makes it harder to edit sections independently in the future if the need arises. Fine leaving it in though.
